metadata-ingestion/source_docs/tableau.md

```diff
@@ -439,19 +489,11 @@ sink:
 *Tableau may not provide schema name when ingesting Custom SQL data source. Use `default_schema_map` to provide a default
 schema name to use when constructing a table URN.
 
+</details>
 
-### Authentication
-
-Currently, authentication is supported on Tableau using username and password
-and personal token. For more information on Tableau authentication, refer to [How to Authenticate](https://help.tableau.com/current/api/metadata_api/en-us/docs/meta_api_auth.html) guide.
-
-
-## Compatibility
-
-Works with Tableau Server Version 2021.1.10 and above. It may also work for older versions.
 
+## Troubleshooting
 
-## Questions
```

Comment:
Can we also add the caveat with Tableau returning wrong information that you had mentioned recently alongwith the document link?

Reply:
Done